.github/workflows/validate-submodule-pull-request.yml

```diff
@@ -1,8 +0,0 @@
-name: Validate Pull Request
```

Comment:
Since this repo is not a part of OSOCE this is not requred I guess.

Reply:
Good point.